src/lib/storage.js

```diff
@@ -10,6 +13,8 @@ class Storage extends ScratchStorage {
     constructor () {
         super();
         this.cacheDefaultProject();
+        this.projectHost = DEFAULT_PROJECT_SERVER;
+        this.assetHost = DEFAULT_ASSET_SERVER;
```

Comment:
Is there a reason not to do this? In scratch-gui code, `setAssetHost` and `setProjectHost` will be called and overwrite this value (with the same value, in the scratch-gui case).

The standalone player could explicitly set these values as well, but having a default lowers the start up cost/boiler plate code for the standalone player scenario.

Reply:
This makes it easier. I think though its one of those slippery slopes kind of deals. We probably don't want a default configuration for scratch that is distributed over Scratch's implementation?